cmd/crossbuild.go

```diff
@@ -136,8 +148,12 @@ func runCrossbuild() {
 	}
 
 	if MIPSPlatformsParam := strings.Join(MIPSPlatforms[:], " "); MIPSPlatformsParam != "" {
```

Comment:
Uppercase variables are exported. Internal variables should not be exported, so they should not start with uppercase. The ARM variable should be adjusted as well `armPlatforms`.

Alternatively, change the variable name and start with something else in lowercase. For example `paramARMPlatform` if you prefer using all caps for the platform name.